packaging/windows/build_pypi_pkg/setup.py

```diff
@@ -22,16 +22,16 @@
 
 class install_torch(setuptools.command.install.install):
     def run(self):
-        if sys.maxsize.bit_length() == 31:
-            raise UserWarning("We don't support Python x86." \
-                "Please install Python x64 instead.")
-
         if sys.platform == 'win32':
             raise UserWarning("Please install using `python "\
                 "-m pip install xxxx -f https://download.pytorch.org/whl/torch_stable.html`.")
 
+        if sys.maxsize.bit_length() == 31:
```

Comment:
Sorry, I misunderstood your previous comment. If it is a Windows-only package, then you shouldn't swap the order here. Determining x86 / x64 Python environment first is more user-friendly.

Reply:
Will remove the sys.platform condition since we do not need it. So only 64bit python detection will happen.